sdk/servicebus/azure-servicebus/azure/servicebus/aio/_base_handler_async.py

```diff
@@ -69,6 +93,17 @@ def __init__(
         self._auth_uri = None
         self._properties = create_properties(self._config.user_agent)
 
+    @classmethod
+    def _convert_connection_string_to_kwargs(cls, conn_str, **kwargs):
+        # pylint:disable=protected-access
+        return BaseHandlerSync._convert_connection_string_to_kwargs(conn_str, **kwargs)
+
+    @classmethod
+    def _create_credential_from_connection_string_parameters(cls, token, token_expiry, policy, key):
+        if token and token_expiry:
+            return ServiceBusSASTokenCredential(token, token_expiry)
+        return ServiceBusSharedKeyCredential(policy, key)
```

Comment:
not blocking: same suggestion as the above, moving to utils might help improve code reusability a bit.